srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -36,30 +39,32 @@
      * @throws NullPointerException if any argument is null
      * @throws IllegalArgumentException if the given expression is not parsable
      */
-    public ValueMatcher(final Variable variable, final String booleanExpression)
+    public ValueMatcher(final Variable variable, final String booleanExpression, final boolean applyMatcher)
```

Comment:
On the UI we're calling this "apply condition" now, so let's update the variable names to be consistent.